pymeasure/instruments/keithley/keithley2602B.py

```diff
@@ -80,44 +109,41 @@ def get_trigger_event_description_strings():
 
 
 class Keithley2600DigitalIOPin:
-    def __init__(self, instrument, pin_number):
+    def __init__(self, instrument: Keithley2602B, pin_number: int) -> None:
         self.instrument = instrument
         self.pin_number = pin_number
 
-    def ask(self, cmd):
+    def ask(self, cmd: str) -> Union[str, None]:
         return self.instrument.ask(f"print(digio.trigger[{self.pin_number}].{cmd})")
 
-    def write(self, cmd):
+    def write(self, cmd: str):
```

Comment:
```suggestion
    def write(self, cmd: str) -> None:
```
And below as well, I think, if you want to be explicit.